index.js

```diff
@@ -20,6 +20,13 @@ module.exports = {
   extends: ['skyscanner', 'prettier', 'prettier/react'],
   plugins: ['prettier'],
   rules: {
+    'import/order': [
+      'error',
+      {
+        groups: ['builtin', 'external', 'parent', 'sibling', 'index'],
```

Comment:
The groups here are just an initial proposal, this is the default order in eslint: https://github.com/benmosher/eslint-plugin-import/blob/master/docs/rules/order.md#groups-array

The order in airbn is a bit different, although they have a `TODO` there to make it more strict:
https://github.com/airbnb/javascript/blob/d951220399c75099838fc79577033b00530b2b90/packages/eslint-config-airbnb-base/rules/imports.js#L146

Reply:
Do you have an example of a before & after? 😇

Reply:
Before:

```
import React from 'react';
import BpkBadge, { BADGE_TYPES } from 'bpk-component-badge';
import { BpkSpinner, SPINNER_TYPES } from 'bpk-component-spinner';
import BpkLoadingButton from 'bpk-component-loading-button';
import {
  InlineTextLg,
  InlineTextSm,
  Paragraph,
  ParagraphLg,
  ParagraphSm,
} from '../../Common/Text';
import HotelStars from '../../HotelStars';
import Distance from '../../Distance';
import Quoted from './Quoted';
import LabelledIcon from './LabelledIcon';
import topAmenities from './top-amenities';
import { withI18n } from '../../i18n-context';
import CardImage from './CardImage';
import MostPopularWith from './MostPopularWith';
import CardLayout from './CardLayout';
import ScoreCircle from './ScoreCircle';
import RatingSummary from './RatingSummary';
import { HotelCardShapeContent } from '../../shapes';
import { FlightsCugBadge } from '../../FlightsCug';
import STYLES from './HotelCard.css';
```

After:
```
import React from 'react';
import BpkBadge, { BADGE_TYPES } from 'bpk-component-badge';
import { BpkSpinner, SPINNER_TYPES } from 'bpk-component-spinner';
import BpkLoadingButton from 'bpk-component-loading-button';

import {
  InlineTextLg,
  InlineTextSm,
  Paragraph,
  ParagraphLg,
  ParagraphSm,
} from '../../Common/Text';
import HotelStars from '../../HotelStars';
import Distance from '../../Distance';
import { withI18n } from '../../i18n-context';
import { HotelCardShapeContent } from '../../shapes';
import { FlightsCugBadge } from '../../FlightsCug';

import Quoted from './Quoted';
import LabelledIcon from './LabelledIcon';
import topAmenities from './top-amenities';
import CardImage from './CardImage';
import MostPopularWith from './MostPopularWith';
import CardLayout from './CardLayout';
import ScoreCircle from './ScoreCircle';
import RatingSummary from './RatingSummary';
import STYLES from './HotelCard.css';
```

Maybe not a huge improvement, but when there're a lot of imports I think it helps to make it more readable. Also, to understand what's is global, what's is defined in a parent and what's local 😅

Reply:
Is it possible to add some alphabetical order as well? And would be good to have css in a separate block too.

Reply:
Maybe alphabetical order can be achieved with https://eslint.org/docs/rules/sort-imports
Not sure about css/scss, I'll investigate both, as it might be good also 👍

Reply:
`sort-imports` does not fix automatically all issues, so I don't think it's a good idea to add this

> The --fix option on the command line automatically fixes some problems reported by this rule: multiple members on a single line are automatically sorted (e.g. import { b, a } from 'foo.js' is corrected to import { a, b } from 'foo.js'), but multiple lines are not reordered.